source/mqtt_broker_evaluation.adoc

```diff
@@ -0,0 +1,51 @@
+
+Technologies investigated:
+
+* Mosquitto
+* RabbitMQ
+* nats jetstream + mqtt enabled
+* HiveMQ
+* EMQX
+* VerneMQ
+
+Mosquitto, HiveMQ and EMQX was disregarded due to lack of needed features or such features beeing behind a paywall.
```

Comment:
It would still be nice to have these features mentioned:
```suggestion
Requirements:

* req 1
* req 2
* ...

Mosquitto, HiveMQ and EMQX was disregarded due to lack of .. or .. being behind a paywall.
```